_data/settings.yml

```diff
@@ -119,6 +119,7 @@ google:
     tag_manager: GTM-TDJ7LGH
 # Shema.org settings used in the schema.html include.
 schema:
+    enabled: false
```

Comment:
What is the purpose of this setting?

Reply:
Toggles the output of schema on the sites - haven't yet fully implemented this feature so it is set to false for now (I'm doing it for linaro.org)